### PR TITLE
module 2 : Contact form structure and Validation

### PR DESCRIPTION
**Pull request by arnold-O**
The name field accepts up to 30 chars.
The email field accepts only emails.
The message field accepts up to 500
Added basic CSS validations 
Added FORMSPREE 3rd party library to collect data from the form.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -494,6 +499,33 @@ body {
...
+  color: #1a2236;
+  background-color: #e5e5e5;
+  box-shadow: inset 0 0 0 0.5px #1a2236;
+}
+
+.form-btn:visited {
+  background: #e5e5e5;
+  box-shadow: inset 0 0 0 0.5px #1a2236;
+  color: #1a2236;
+}
```
Comment (PinkMoon25) [inline style.css:527]: - [ ] Great work implementing the contact form :clap: , however please notice that as mentioned in the project requirements **"In order to lay out the elements on the page you should use Flexbox."** I would suggest to please use flexbox for the contact form section and even for the form's layout
Flexbox is an essential skill that we have to master as a front-end developer, we need you to be a flex ninja :ninja: It gives you more control over the layout and is totally responsive so please use it as much as you can :pray:

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @arnold-O , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
Form implementation :heavy_check_mark: 
HTML validation :heavy_check_mark: 
Integrate formspree :heavy_check_mark: 
Good Readme :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by laiifuu — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

## Hi Team :wave:  

Great work on the project so far 👏 👏
However, there are still some issues you need to address in order to move on to the next one.
so-close


 ## Highlights
- [x] Great design :fire: 

 ## Required Changes ♻️
- [ ] Your UI is looking great :clap: Nice job! However, as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), please use flexbox to align your inputs inside the form. I noticed that you used margins to align your inputs. You can have the same result if you use flexbox. Adding the following CSS declaration to your style sheet after removing any margin-top to the inputs should do the trick: 
`form {
    display: flex;
    flex-direction: column;
    gap: 8px;
}
`

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and :computer: Happy coding! :computer:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arnold-O** [comment]
> # Status: Changes Required ♻️
> ## Hi Team 👋
> Great work on the project so far 👏 👏 However, there are still some issues you need to address in order to move on to the next one. so-close [ so-close ](https://user-images.githubusercontent.com/93520178/212988842-a2a9a057-d6fb-4149-b6e5-67b91ef1281b.gif) [ ](https://user-images.githubusercontent.com/93520178/212988842-a2a9a057-d6fb-4149-b6e5-67b91ef1281b.gif)
> 
> ## Highlights
> * [x]  Great design 🔥
> 
> ## Required Changes ♻️
> * [ ]  Your UI is looking great 👏 Nice job! However, as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), please use flexbox to align your inputs inside the form. I noticed that you used margins to align your inputs. You can have the same result if you use flexbox. Adding the following CSS declaration to your style sheet after removing any margin-top to the inputs should do the trick:
>   `form { display: flex; flex-direction: column; gap: 8px; } `
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and 💻 Happy coding! 💻
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @arnold-O, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Abdessittir** [comment]
Hi @arnold-O, You submitted this approved pull request for the desktop version project,
Kindly create a new repo for this project and submit it

**Pull request merged**